services/api-server/tests/unit/api_studies/test_api_routes_studies_jobs.py

```diff
@@ -225,3 +227,62 @@ def _default_side_effect(
         },
     )
     assert response.status_code == 200
+
+
+async def test_get_study_job_outputs(
+    client: httpx.AsyncClient,
+    fake_study_id: UUID,
+    auth: httpx.BasicAuth,
+    mocked_webserver_service_api_base: MockRouter,
+):
+    job_id = "cfe9a77a-f71e-11ee-8fca-0242ac140008"
+
+    capture = {
+        "name": "GET /projects/cfe9a77a-f71e-11ee-8fca-0242ac140008/outputs",
+        "description": "<Request('GET', 'http://webserver:8080/v0/projects/cfe9a77a-f71e-11ee-8fca-0242ac140008/outputs')>",
+        "method": "GET",
+        "host": "webserver",
+        "path": {
+            "path": "/v0/projects/{project_id}/outputs",
+            "path_parameters": [
+                {
+                    "in": "path",
+                    "name": "project_id",
+                    "required": True,
+                    "schema": {
+                        "title": "Project Id",
+                        "type": "str",
+                        "pattern": None,
+                        "format": "uuid",
+                        "exclusiveMinimum": None,
+                        "minimum": None,
+                        "anyOf": None,
+                        "allOf": None,
+                        "oneOf": None,
+                    },
+                    "response_value": "projects",
+                }
+            ],
+        },
+        "query": None,
+        "request_payload": None,
+        "response_body": {"data": {}},
+        "status_code": 200,
+    }
+
+    mocked_webserver_service_api_base.get(
+        path=capture["path"]["path"].format(project_id=job_id)
+    ).respond(
+        status_code=capture["status_code"],
+        json=capture["response_body"],
+    )
```

Comment:
I would encourage you to use the fixture I added for this (see [here](https://github.com/bisgaard-itis/osparc-simcore/blob/master/services/api-server/tests/unit/api_studies/test_api_routes_studies_jobs.py#L209)). I think (or at least hope 🤞🏻) it should be easier than doing this stuff by hand.

Reply:
Yes, I saw it. For this simple case I did a shortcut but I will give it another try with yours